pkg/networkplugin-syncer/handlers/ovn/ovn_submariner_infrastructure.go

```diff
@@ -141,12 +141,12 @@ func (ovn *SyncHandler) ensureSubmarinerInfra() error {
 func (ovn *SyncHandler) setupOvnClusterRouterLRPs() error {
 	remoteSubnets := ovn.remoteEndpointSubnetSet()
 
-	if remoteSubnets.Size() == 0 {
+	if len(remoteSubnets) == 0 {
```

Comment:
Why not use `remoteSubnets.Len()` here?

Reply:
They’re equivalent (currently), but I see from https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/util/sets/set_generic_test.go that the API does appear to be `s.Len()` rather than `len(s)` (which makes sense since the former is controlled by the implementation, and the latter is a happy coincidence on the current implementation).